examples/lifelong_learning/atcii/README.md

```diff
@@ -0,0 +1,163 @@
+# Using Lifelong Learning Job in Thermal Comfort Prediction Scenario
+
+This document introduces how to use lifelong learning job in thermal comfort prediction scenario. 
+Using the lifelong learning job, our application can automatically retrains, evaluates, 
+and updates models based on the data generated at the edge.
+
+##  Thermal Comfort Prediction Experiment
+
+
+### Install Sedna
+
+Follow the [Sedna installation document](/docs/setup/install.md) to install Sedna.
+
+### Prepare Dataset
+In this example, you can download ASHRAE Global Thermal Comfort Database II to initial lifelong learning Knowledgebase .
+
+
+
+download [datasets](https://kubeedge.obs.cn-north-1.myhuaweicloud.com/examples/atcii-classifier/dataset.tar.gz), including train, evaluation and incremental dataset.
+```
+cd /
+wget https://kubeedge.obs.cn-north-1.myhuaweicloud.com/examples/atcii-classifier/dataset.tar.gz
+tar -zxvf dataset.tar.gz
+```
+
+### Create Lifelong Job
+in this example, `$WORKER_NODE` is a custom node, you can fill it which you actually run.
+
+
+```
+WORKER_NODE="edge-node" 
+```
+Create Dataset
+
+```
+kubectl create -f - <<EOF
+apiVersion: sedna.io/v1alpha1
+kind: Dataset
+metadata:
+  name: lifelong-dataset
+spec:
+  url: "/data/lifelong_learning/trainData.csv"
+  format: "csv"
+  nodeName: $WORKER_NODE
+EOF
+```
+
+Also, you can trigger retraining by use `incremental Dataset`[trainData2.csv] to replace `trainData.csv` 
+
+Start The Lifelong Learning Job
+
+```
+
+kubectl create -f - <<EOF
+apiVersion: sedna.io/v1alpha1
+kind: LifelongLearningJob
+metadata:
+  name: atcii-classifier-demo
+spec:
+  dataset:
+    name: "lifelong-dataset"
+    trainProb: 0.8
+  trainSpec:
+    template:
+      spec:
+        nodeName: "edge-node"
```

Comment:
pls use $WORKER_NODE instead of edge-node

Reply:
fix